From 57699052e6eb16293a5954ad4312331448d9ef10 Mon Sep 17 00:00:00 2001
From: Jim Stichnoth <jstichnoth@mythtv.org>
Date: Wed, 16 Jul 2014 13:54:46 -0700
Subject: [PATCH 1/2] Subtitles: Fix cc608 indents.

The original code was trying to be clever and indent one character
less than intended presumably to account the extra space character
that control codes also imply, but it's not clear that was ever an
appropriate thing to do.

Thanks to faginbagin@ for identifying the problem and the fix.
Refs #12051.
(cherry picked from commit 7d60440b2d790208a25ce707480f5b01657410f1)
---
 mythtv/libs/libmythtv/cc608decoder.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythtv/cc608decoder.cpp b/mythtv/libs/libmythtv/cc608decoder.cpp
index e055718..f9768c1 100644
--- a/mythtv/libs/libmythtv/cc608decoder.cpp
+++ b/mythtv/libs/libmythtv/cc608decoder.cpp
@@ -844,7 +844,7 @@ int CC608Decoder::NewRowCC(int mode, int len)
     lastrow[mode] = newrow[mode];
     newrow[mode] = 0;
 
-    int limit = (newattr[mode] ? newcol[mode] - 1 : newcol[mode]);
+    int limit = newcol[mode];
     for (int x = 0; x < limit; x++)
     {
         ccbuf[mode] += ' ';
-- 
1.7.10.2

